crates/locutus-core/src/client_events.rs

```diff
@@ -349,9 +352,9 @@ impl ClientRequest<'_> {
     }
 
     /// Deserializes a `ClientRequest` from a MessagePack encoded request.  
-    pub fn decode_mp(msg: &[u8]) -> Result<Self, ErrorKind> {
+    pub fn decode_mp(msg: &[u8]) -> Result<Self, WsApiError> {
```

Comment:
we should be implementing the new decoding trait for ContractResponse instead and using it instead of the this static method

Reply:
Done